### DISCUSSION TITLE
Front End - Azurechat Accelerator

### DISCUSSION THREAD

**Discussion by keith-oak** [microsoft/PubSec-Info-Assistant#335] [Ideas]
Just a question - any chance of front-ending this with the nicer UI and components from Azurechat - seems like a missed opportunity?

**Comment by dayland** [comment]
I am unaware of Azurechat... can you share a link?

**Reply by eric-sf** [reply]
https://github.com/microsoft/azurechat

**Reply by dayland** [reply]
At this time there is not a plan to update the UX to the one from Azurechat.

**Comment by lon-tierney** [comment]
Closing discussion as this is not in roadmap. Still optional for others to implement in another branch/fork.
